tests/manage/z_cluster/nodes/test_node_replacement_proactive.py

```diff
@@ -284,6 +285,12 @@ def test_nodereplacement_proactive(self):
             timeout=1800
         ), "Data re-balance failed to complete"
 
+        ocs_version_semantic = version.get_semantic_ocs_version_from_config()
+        if ocs_version_semantic >= version.VERSION_4_13:
+            assert (
+                verify_only_ocs_nodes_in_storagecluster
+            ), "Not only nodes with OCS label in storagecluster"
```

Comment:
Storagecluster node topology is having an entry of non ocs node(s)

Reply:
done